cypress/e2e/test/invalidlogin.cy.js

```diff
@@ -0,0 +1,38 @@
+import { invalidEmail, invalidPassword } from '../../support/testdata';
+
+describe('Cannot login with invalid credentials', () => {
+  it('cannot login with invalid credentials', () => {
+    // Visit the login page
+    cy.visit('https://bekzatbagdat.github.io/social-media-client/');
```

Comment:
```suggestion
    cy.visit('/');
```

It is very important to target the right code here. What you are targetting is code that has already been merged to master, you should be looking at the code in your PR that has not yet been merged.

The way to do this is use a local address, since Cypress will run its own dev server for you.